src/igvfd/commands/make_audit_docstring_json.py

```diff
@@ -0,0 +1,70 @@
+import json
+
+import igvfd.audit.biosample
+import igvfd.audit.construct_library
+import igvfd.audit.formatter
+import igvfd.audit.human_donor
+import igvfd.audit.in_vitro_system
+import igvfd.audit.item
+import igvfd.audit.measurement_set
+import igvfd.audit.ontology_term
+import igvfd.audit.sample
+import igvfd.audit.tissue
+import igvfd.audit.treatment
+import igvfd.audit.variant
+import igvfd.audit.whole_organism
+
+
+NON_AUDIT_FUNCTION_NAMES = [
+    'audit_checker',
+    'audit_link',
+]
+
+AUDIT_MODULES_TO_PROCESS = [
+    igvfd.audit.biosample,
+    igvfd.audit.construct_library,
+    igvfd.audit.formatter,
+    igvfd.audit.human_donor,
+    igvfd.audit.in_vitro_system,
+    igvfd.audit.item,
+    igvfd.audit.measurement_set,
+    igvfd.audit.ontology_term,
+    igvfd.audit.sample,
+    igvfd.audit.tissue,
+    igvfd.audit.treatment,
+    igvfd.audit.variant,
+    igvfd.audit.whole_organism,
+]
+
+
+def get_audit_function_names_from_module(module):
+    module_attributes = dir(module)
+    audit_function_names = [attribute for attribute in module_attributes if attribute.startswith(
+        'audit_') and attribute not in NON_AUDIT_FUNCTION_NAMES]
+    return audit_function_names
+
+
+def get_docstring_dict_from_function_name(function_name):
+    docstring = eval(function_name + '.__doc__')
+    if docstring is not None:
+        return {function_name: docstring.strip()}
+    else:
+        return {function_name: ''}
+
+
+def main():
+    audit_function_full_names = []
+    for audit_module in AUDIT_MODULES_TO_PROCESS:
+        module_name = audit_module.__name__
+        function_names_in_module = get_audit_function_names_from_module(audit_module)
+        for function_name in function_names_in_module:
+            audit_function_full_names.append(module_name + '.' + function_name)
+    audit_docstring_dict = {}
+    for function_name in audit_function_full_names:
+        audit_docstring_dict.update(get_docstring_dict_from_function_name(function_name))
+    with open('/static/json/auditdoc.json', 'w') as audit_json:
```

Comment:
Can you just dump this into the `/igvfd/src/static/...` folder instead? Then the existing static route would just serve it I think: https://github.com/IGVF-DACC/igvfd/blob/cbd653c35060cbd9cc20cc273dea22b670314117/src/igvfd/__init__.py#L42

Reply:
yup.